external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaDataConsumerSuite.scala

```diff
@@ -121,4 +125,158 @@ class KafkaDataConsumerSuite extends SharedSparkSession with PrivateMethodTester
       threadpool.shutdown()
     }
   }
+
+  test("SPARK-25151 Handles multiple tasks in executor fetching same (topic, partition) pair") {
```

Comment:
As we've spoken about this in another PR not sure the jira number helps here (since it's not a bugfix).

Reply:
Hmm... that's actually dealing with the limitation of current pool, and I guess other reviewer was pointing out this issue so I feel we've indicated this as soft kind of bug. Don't mind whether removing JIRA number or not.

Reply:
I've read back the discussion and now see the point. I still think it looks weird and can be added as comment but let's leave this part to the person who will do the merge.